### ISSUE TITLE
`@stylistic/function-parentheses-newline-inside` writes two fixes that are safe apart and destroy the value together

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#312]
Found while fixing #281, and reproduced on `main` byte for byte, so that branch neither adds it nor touches it.

The two guards of `function-parentheses-newline-inside` are asked separately — one about the opening parenthesis, one about the closing — and under `never-multi-line` both fixes run in the same pass over the same value. Each asks what its own fix would leave behind and neither knows of the other, so two writes that are safe apart destroy the value together.

### What minimal example or steps are needed to reproduce the bug?

```less
a { t: foo(1px // c) calc(
/*b*/
); }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
	"customSyntax": "postcss-less",
	"plugins": "@stylistic/stylelint-plugin",
	"rules": {
		"@stylistic/function-parentheses-newline-inside": "never-multi-line"
	}
}
```

### What did you expect to happen?

Either both breaks taken away and a file that still parses, or the problem reported and nothing written.

### What actually happened?

`--fix` writes `a { t: foo(1px // c) calc(/*b*/); }` and reports a clean pass. The comment `// c) calc(/*b*/); }` now runs to the end of the line, the block has nothing to close it, and the next run throws `Unclosed bracket`. Under `postcss-scss` the same.

Each guard was right on its own. The call holds nothing but a comment, so the opening question is asked about the closing parenthesis, which the break in front of it keeps outside the comment; the closing question is asked about that same parenthesis with only the whitespace in front of it gone, and the break behind the opening parenthesis still keeps it outside. Between them every break of the value goes.

### Do you have a proposal to fix the bug?

The two questions have to be asked of one text — the value as both fixes would leave it — rather than of two texts each holding the other fix's whitespace. Where the two fixes are written to one function, that is the union of the stretches either of them empties; where they are written to two, each is still its own question.

A function holding nothing but comments and whitespace is the shape this stands on, since that is where the two questions are about the same character.
